### PR TITLE
[fireblocks] Adjust confirmation threshold

### PR DESCRIPTION
**Pull request by ian-shim**
Allow setting confirmation threshold for fireblocks transactions (which default to 3 otherwise)

### CODE REVIEW

**Comment by mooselumph** [comment]
Seems like this PR sets a fixed confirmation threshold for the wallet and then updates the confirmation threshold for each individual transaction. I suppose there isn't a global fireblocks setting that can be used instead?

**Comment by ian-shim** [comment]
> Seems like this PR sets a fixed confirmation threshold for the wallet and then updates the confirmation threshold for each individual transaction. I suppose there isn't a global fireblocks setting that can be used instead?

There doesn't seem like an option to set the global threshold for the account 😢

File: chainio/clients/wallet/fireblocks_wallet.go [hunk 1]
```diff
@@ -199,6 +201,14 @@ func (t *fireblocksWallet) SendTransaction(ctx context.Context, tx *types.Transa
 	t.txIDToNonce[res.ID] = nonce
 	t.logger.Debug("Fireblocks contract call complete", "txID", res.ID, "status", res.Status)
 
+	success, err := t.fireblocksClient.SetConfirmationThreshold(ctx, res.ID, t.confirmationThreshold)
```
Comment (mooselumph) [inline chainio/clients/wallet/fireblocks_wallet.go:204]: Maybe use `confirmationThreshold=0` or `-1` to just use the default?

**Review by mooselumph — APPROVED** [review]
lgtm

**Pull request closed without merging**